src/DebugCodeLens/DebugCodeLensProvider.ts

```diff
@@ -7,12 +7,13 @@ import { TestReconciliationState } from '../TestResults'
 import { TestState, TestStateByTestReconciliationState } from './TestState'
 import { JestExt } from '../JestExt'
 
+type GetJestExt = (uri: vscode.Uri) => JestExt | undefined

```

Comment:
Since it's the second time defined that way, would it be worth the effort to outsource it (for example to `vscode-jest/JestExt.ts`)?

Reply:
we could,  but it is not a method defined in `JextExt` but from `extensionManager`, so I move the type alias there instead. 

The real concern here is the fact that typescript didn't catch the original `undefined` type error and caused run time crash so I refactor the code and introduced an explicit typedef, in my opinion, a bit unnecessary if tsc does its job right...